src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -378,30 +381,54 @@ def generate_zero_coverage_report(self):
         with open('code-coverage-reports/zero_coverage_functions.json', 'w') as f:
             json.dump(zero_coverage_function_counts, f)
 
-    def generate_files_list(self, covered=True, chunk=None):
+    def generate_files_list(self, covered=True, platform=None, chunk=None):
         options = ['--filter-covered', '--threads', '2']
-        files = self.generate_info(chunk=chunk, out_format='files', options=options)
+        files = self.generate_info(platform=platform, chunk=chunk, out_format='files', options=options)
         return files.splitlines()
 
     def generate_chunk_mapping(self):
-        def get_files_task(chunk):
-            return lambda: (chunk, self.generate_files_list(True, chunk=chunk))
+        def get_files_task(platform, chunk):
+            return lambda: (platform, chunk, self.generate_files_list(True, platform=platform, chunk=chunk))
 
         with ThreadPoolExecutor(max_workers=4) as executor:
             futures = []
-            for chunk in self.get_chunks():
-                futures.append(executor.submit(get_files_task(chunk)))
+            for platform in ['linux', 'windows']:
+                for chunk in self.get_chunks():
+                    futures.append(executor.submit(get_files_task(platform, chunk)))
 
-            with sqlite3.connect('chunk_mapping.db') as conn:
+            with sqlite3.connect('chunk_mapping.sqlite') as conn:
                 c = conn.cursor()
-                c.execute('CREATE TABLE files (path text, chunk text)')
+                c.execute('CREATE TABLE file_to_chunk (path text, platform text, chunk text)')
+                c.execute('CREATE TABLE chunk_to_test (platform text, chunk text, path text)')
 
                 for future in concurrent.futures.as_completed(futures):
-                    (chunk, files) = future.result()
-                    c.executemany('INSERT INTO files VALUES (?,?)', [(f, chunk) for f in files])
+                    (platform, chunk, files) = future.result()
+                    c.executemany('INSERT INTO file_to_chunk VALUES (?,?,?)', ((f, platform, chunk) for f in files))
+
+                # Retrieve chunk -> tests mapping from ActiveData.
+                r = requests.post('https://activedata.allizom.org/query', data=json.dumps({
```

Comment:
Don't you want to use a TC secret here to specify different urls for staging/prod ?

Reply:
ActiveData has no staging/production difference, so we will always use the same URL.

Reply:
(The `allizom` URL is confusing, but it's actually the only one)